README.rst

```diff
@@ -21,10 +21,11 @@ instructions <DEVELOPMENT.rst>`_.
 Get OpenSlides
 --------------
 
-First, you have to clone this repository::
+First, you have to clone this repository, then checkout to the latest stable branch::
 
     git clone https://github.com/OpenSlides/OpenSlides.git --recurse-submodules
     cd OpenSlides/
+    git checkout remotes/origin/stable/3.4.x
```

Comment:
Nitpick, but I think
```suggestion
    git checkout stable/3.4.x
```
should be enough, shouldn't it?

Reply:
That's one thing, but the bigger issue at the moment is that some tests fail (python installation, dependencies) even with minor modifications.